vault/logical_system_helpers.go

```diff
@@ -74,6 +74,13 @@ var (
 					Pattern:    pattern,
 					Operations: make(map[logical.Operation]framework.OperationHandler),
 					Fields:     make(map[string]*framework.FieldSchema),
+					DisplayAttrs: &framework.DisplayAttributes{
+						// Since we lack full information for Fields, and all information for Responses, the generated
+						// OpenAPI won't be good for much other than identifying the endpoint exists at all. Thus, it
+						// is useful to make it clear that this is only a stub. Code generation will use this to ignore
+						// these operations.
+						OperationPrefix: "enterprise-stub-system",
```

Comment:
One quick thought. Would it make sense to call this "enterprise-stub" instead? For other prefixes under `/sys`, we've been dropping the "system" part. It's not a big deal regardless since it will be filtered out, just a thought.

Reply:
The `system-` prefix had not been dropped for any of these, so I was just prepending `enterprise-stub-` to what was already there. But, sure, might as well bring these into line with the surrounding convention as well.

In that case, I'll also fix `sys/replication/status` which is not a stub, but hasn't had the `system` part dropped yet.